Riot/Modules/Analytics/Analytics.swift

```diff
@@ -275,8 +275,16 @@ extension Analytics {
         let event = AnalyticsEvent.Error(
             context: context,
             cryptoModule: .Rust,
+            cryptoSDK: AnalyticsEvent.Error.CryptoSDK.Rust,
```

Comment:
It should be fine to infer the type here like the others do.
```suggestion
            cryptoSDK: .Rust,
```